src/sonar.ts

```diff
@@ -41,20 +41,20 @@ interface SonarCloudInputs {
 
 export function getSonarCloudInputs(): SonarCloudInputs {
   const apiUrl = core.getInput("sonar-api-url", { required: true });
-  const token = core.getInput("sonar-token", { required: true });
+  const token = core.getInput("sonar-token", { required: false });
```

Comment:
Si this change affecting anything in the behaviour of the action?

Reply:
No, it's not mandatory for public repositories. It was a mistake that I noticed